content/gherkin/reference.md

```diff
@@ -330,6 +331,60 @@ to match the step against a step definition.
 
 You can also use parameters in [multiline step arguments](#step-arguments).
 
+## Rule
+
+The `Rule` entity is a rather new idiom in Gherkin (since Gherkin v6).
+Each `Rule` is intended to represent one *business rule* that should be implemented.
+A `Rule` is used to group together several `Example` (aka: `Scenario`) / `Scenario Outline`
+that belong to this *business rule*. In addition, a `Rule` may contain a `Background` section.
+
+Rule parent:
+
+- `Feature` (cardinality: one)
+
+Rule children:
+
+- `Description` (cardinality: optional, `0 .. 1`)
```

Comment:
Do we need to mention the cardinality? Does that clarify things or would it be a possible source of confusion?

Reply:
You are basically describing a grammar in your own words.
You can leave it out. But then it is no longer a (readable) specification. 
Meaning, how do you understand how man grammar-elements may occur.

@mlvandijk 
Just decide.